src/Pages/Home/Home.jsx

```diff
@@ -0,0 +1,81 @@
+import { Navbar } from "../../Components/Navbar/Navbar";
+import "./home.css";
+import { Link } from "react-router-dom";
+import { categories } from "../../backend/db/categories";
+const Home = () => {
+  return (
+    <div>
+      <Navbar />
+      {/* From Banner */}
+      <div>
+        <div
+          className="home-bg"
+          style={{ backgroundImage: "url('Assets/ecom-bg.jpg')" }}
+        >
+          <div className="home-text-section">
+            <h1 className="home-heading">Shop with Quality and Trust</h1>
+            <button className="shop-btn">
+              <Link className="shop-link-btn" to="/productlisting">
+                Shop Now{" "}
```

Comment:
is there any specific reason for having these empty curly braces, bcz if not you can remove them!!

Reply:
No , thats done by the Code Formatter , I'll remove that